constructorio-client/src/main/java/io/constructor/client/models/Task.java

```diff
@@ -0,0 +1,96 @@
+package io.constructor.client.models;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Constructor.io Task ... uses Gson/Reflection to load data in
+ */
+public class Task {
+
+    @SerializedName("id")
+    private int id;
+
+    @SerializedName("type")
+    private String type;
+
+    @SerializedName("status")
+    private String status;
+
+    @SerializedName("submission_time")
+    private String submissionTime;
+
+    @SerializedName("last_update")
+    private String lastUpdate;
+
+    @SerializedName("start_time")
+    private String startTime;
+
+    @SerializedName("filename")
+    private String filename;
+
+    @SerializedName("protocol")
+    private String protocol;
+
+    @SerializedName("result")
+    private Object result;
+
+    /**
+     * @return the id
+     */
+    public int getId() {
+        return id;
+    }
+
+    /**
+     * @return the type
+     */
+    public String getType() {
+        return type;
+    }
+
+    /**
+     * @return the status
+     */
+    public String getStatus() { return status; }
```

Comment:
Let's keep the function blocks consistent with functions above. Same with the fn below